test/unit/org/transmartproject/export/SNPMockDataHelper.groovy

```diff
@@ -0,0 +1,158 @@
+package org.transmartproject.export
+
+import grails.test.mixin.*
+
+import org.transmartproject.core.dataquery.DataRow
+import org.transmartproject.core.dataquery.Patient
+import org.transmartproject.core.dataquery.TabularResult
+import org.transmartproject.core.dataquery.assay.Assay
+import org.transmartproject.core.dataquery.highdim.AssayColumn
+import org.transmartproject.db.dataquery.highdim.snp_lz.SnpLzCell
+import org.transmartproject.db.dataquery.highdim.snp_lz.SnpLzRow
+import org.transmartproject.db.dataquery.highdim.snp_lz.SnpSubjectSortedDef
+
+class SNPMockDataHelper extends MockTabularResultHelper {
```

Comment:
That's the mock data generator, not the test (does not contain methods annotated with @Test)